python-avocado.spec

```diff
@@ -172,6 +172,9 @@ popd
 pushd optional_plugins/runner_docker
 %{__python} setup.py develop --user
 popd
+pushd optional_plugins/resultsdb
+%{__python} setup.py develop --user
```

Comment:
Actually does this works properly? I'm seeing this:

```
test_output_incompatible_setup (selftests.functional.test_output.OutputPluginTest) ... ok
test_output_incompatible_setup_2 (selftests.functional.test_output.OutputPluginTest) ... skipped 'Uncapable of Avocado Result HTML plugin'
ok
test_redirect_output (selftests.functional.test_output.OutputPluginTest) ... ok
```
which looks like despite we develop `html` plugin it's either not enabled or malfunctioning...